lib/bundler_api/web.rb

```diff
@@ -273,6 +273,24 @@ def get_cached_dependencies
     dependencies
   end
 
+  def get_cached_info(name)
+    key = "info/#{name}"
+    @dalli_client.fetch(key) do
+      calc_info = @gem_info.info(name)
+      @dalli_client.set(key, calc_info)
```

Comment:
setting it manually is unnecessary (http://www.rubydoc.info/gems/dalli/2.7.6/Dalli%2FClient%3Afetch)